internal/handlers/common/error.go

```diff
@@ -40,6 +40,7 @@ const (
 	ErrProjectionInEx    = ErrorCode(31253) // Location31253
 	ErrProjectionExIn    = ErrorCode(31254) // Location31254
 	ErrRegexOptions      = ErrorCode(51075) // Location51075
+	ErrIllegalKey        = ErrorCode(15974) // Location15974
```

Comment:
1. suggest format of the errors:  `Err <Entity Name> <error details related to this entity>`
2. use of this error in context when the sort type is wrong, can it be renamed to smth like `err sort type wrong`?

Reply:
I think `ErrSortBadValue` would be ok as it's about bad sort value

Reply:
It is.